docs/src/environments/_index.md

```diff
@@ -1,51 +1,86 @@
 ---
 title: "Manage Platform.sh environments"
 weight: -75
+layout: single
 sidebarTitle: Manage environments
 description: Learn what environments on Platform.sh are and how to take advantage of them.
 ---
 
-On Platform.sh, an environment is a logically separate instance of an app or group of apps
-with all of the services needed to run the app.
+A Platform.sh environment contains one instance of an app (or [group of apps](../create-apps/multi-app.md))
+with all the services needed for it to run.
 
 You can think of an environment as a complete working website
-that's related to but safely isolated from others in the project.
+that's related to but safely isolated from others in your project.
 So you can run tests and review a complete working copy without worrying about damaging anything in production.
 
 Each project includes multiple environments,
 often divided into [environment types](../administration/users.md#environment-types).
 If you have a live site, you have at least a production environment.
 You may also have additional environments for development, testing, staging, review, and so on.
+To organize your environments, you can create [hierarchical relationships](#hierarchy).
```

Comment:
:+1: I like the move here. Makes sense right after listing how many there can be.